minishift/openshift/f8notification.app.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: v1
+kind: Template
```

Comment:
here you used a template, but what's the benefit compared to a `DeploymentConfig`?

Reply:
If you see you can have few parameters here like `IMAGE_NAME`, `TAG`, `MEMORY`. This way it's easy for a user to just overwrite this parameter using `-p` flag.

Reply:
ok